swift-sdk/Internal/IterableUtil.swift

```diff
@@ -29,6 +29,10 @@ import os
         return !isNullOrEmpty(string: string)
     }
     
+    static func generateUUID() -> String {
+        return UUID().uuidString.replacingOccurrences(of: "-", with: "")
```

Comment:
Any particular reason you wanted to remove the "-" in the UUID?
What if we need to obtain hashValue of the UUID?

Reply:
It is just a matter of preference. I don't think it matters as long as the values are unique. I will change it so that we don't replace the '-'s anymore.

Since the generated UUID is a string. The hashValue will have the same semantics as String so we should be fine.

Reply:
I was thinking if we used the built in HashValue function it probably uses the "-" as part of the string to hash. UUIDs are pretty standard with the "-", so it might be simpler to leave it in.